schedule.md

```diff
@@ -6,19 +6,41 @@ Always be commiting. Deploy early and often.
 
 Here's a rough sketch of what you should do and in what order:
 
-1.  [ ] Sketch some rough wireframes for how the front end will look and act.
-1.  [ ] Write out some user stories for the app. User stories are short of how a
-    user interacts with your app, and follows the format "As a `<role>`, I want
-    to `<do something>`, so that `<some goal>`.
-1.  [ ] Model the entities in your app. Draw a diagram. Use your wireframes and
-    user stories to drive you modeling process by asking "What 'things' are a
-    user interacting with?"
-1.  [ ] Create a repo that your project will use, and add a README to it.
-1.  [ ] Create a simple front-end with HTML and CSS, and host the front end on
-    GitHub Pages. Use your wireframes to guide your layout.
-1.  [ ] Create the code to manage your game logic.
```

Comment:
I really want to emphasize that folks should be writing their engine first. If they can't write their engine first, it's ok to make a clickable board.

However, separation of concerns is easiest when starting with the engine and I want to encourage that.

And the engine really is a pre-requisite for game resource AJAX.